docs/DocumentationStyleGuide/accessibility.rst

```diff
@@ -0,0 +1,94 @@
+Accessibility
+=============
+
+The moja global community strives towards accessibility in our software
+products and even in the documentation. Through accessibility, we strive
+for successful access to information and resources among people who
+might find it challenging to access them otherwise. We aim to bring
+contributors with limited ability to the forefront of our community to
+have them contribute and access our documentation easily.
+
+The general guidelines for creating accessible documentation pieces are:
+
+-  Maintain consistency across the documentation. Both in style and
+   visuals by employing a fixed hierarchy.
+-  Avoid language or word usage that reeks of bias or a disregard for a
+   group of people.
+-  Focus on the audience that you are writing for and follow a
+   conversation style of documentation.
+-  Define acronyms, abbreviations, prerequisites initially and never
+   assume the technical competency of the reader.
+-  Use alternative text for multimedia elements wherever possible, and
+   be conscious of the usage of visual elements.
+
+Image accessibility
+-------------------
+
+-  Validate that the images, diagrams, graphs, icons possess a
+   meaningful alternative-text description for a meaningful equivalent.
+-  Use alternative text to describe what the image, digraph, graph or
+   icon does, rather than what it looks like.
+-  All images, diagrams, graphs, icons should contain a short image
+   description that is self-explanatory.
+-  Prefer the usage of SVGs over JPEGs and PNGs. SVGs scale on zooming
+   and provide better context to the readers.
+-  Don’t use images of text, terminal outputs, code samples and other
```

Comment:
THIS! 🚀 

Make it stronger - 'Always transcribe text or terminal outputs as code blocks' or something. Code is hard to read at the best of times, let alone in screenshot thumbnails.